apps/desktop/i2p-router-app/src/main/resources/css/i2p_router_app.css

```
@@ -0,0 +1,227 @@
+    -fx-font-family: "IBM Plex Sans Light";
+}
+
+.error-text {
+    -fx-fill: -bisq-error;
+    -fx-text-fill: -bisq-error;
+    -fx-font-size: 1.1em;
+    -fx-font-family: "IBM Plex Sans Light";
+}
+
+#bisq-i2p-router-logo {
+    -fx-image: url("/images/bisq-i2p-router-logo.png");
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)